src/platform/plugins/shared/vis_types/timeseries/public/convert_to_lens/lib/convert/column.ts

```diff
@@ -107,7 +107,7 @@ export const isColumnWithMeta = (column: Column): column is ColumnWithMeta => {
   return false;
 };
 
-export const excludeMetaFromColumn = (column: Column) => {
+export const excludeMetaFromColumn = <T extends Column>(column: T) => {
```

Comment:
This generic type was introduced to distinguish between a bucket and metric column type correctly